packages/atomic/src/components/atomic-search-interface/lang/dev.json

```diff
@@ -0,0 +1,4 @@
+{
+  "search": "_search_",
+  "clear": "_clear_"
```

Comment:
Shouldn't the values exactly match the keys in this JSON file? Otherwise, I would think I should add a translation using the key `_clear_`.